roles/scaffold_rm_facts/templates/module_utils/network_os/config/resource/resource.py.j2

```diff
@@ -13,6 +13,14 @@ created
 from ansible.module_utils.network.common.cfg.base import ConfigBase
 from ansible.module_utils.network.common.utils import to_list
 from {{ import_path }}.{{ network_os }}.facts.facts import Facts
+{% if transport.netconf %}
+from ansible.module_utils.network.junos.junos import (locked_config,
```

Comment:
This import can be added only in case network_os is junos, as it is applicable for junos only.
For plain netconf it can be changed to 
`from ansible.module_utils.network.netconf.netconf import locked_config`
and other import usages can be replaced with `self._module._connection.<api-name>.`
For eg: `self._module._connection.edit_config()` `self._module._connection.commit()` etc